### PR TITLE
Removes entire page paints on scroll. Fixes #37

### LINKED ISSUE #37: Entire page paints on scroll
screenflow

Adding `transform: translateZ(0)` to the .content region fixes this. core-header-panel also suffers from this issue.

### PR DESCRIPTION
**Pull request by ebidel**
R: @frankiefu

### CODE REVIEW

**Comment by ebidel** [comment]
cc @ssorallen

File: core-scroll-header-panel.css [hunk 1]
```diff
@@ -26,6 +26,11 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
   overflow-y: auto;
 }
 
+/* github.com/Polymer/core-scroll-header-panel/pull/38*/
+#mainContainer {
+  transform: translateZ(0);
```
Comment (ssorallen) [inline core-scroll-header-panel.css]: This should also have a -webkit-transform version before it. Safari, Mobile Safari, and Android Browser still use the prefix: http://caniuse.com/#search=transform

Comment (ebidel) [inline core-scroll-header-panel.css]: Done.

**Comment by ebidel** [comment]
PTAL

File: core-scroll-header-panel.css [hunk 2]
```diff
@@ -26,6 +26,12 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
   overflow-y: auto;
 }
 
+/* github.com/Polymer/core-scroll-header-panel/pull/38*/
+#mainContainer {
```
Comment (ssorallen) [inline core-scroll-header-panel.css]: There's a #mainContainer selector right above this one. Can they be collapsed into one?

Comment (ebidel) [inline core-scroll-header-panel.css]: Done

**Pull request merged**